worker/transcribee_worker/worker.py

```diff
@@ -52,18 +53,13 @@ def _get_tmpfile(self, filename: str) -> Path:
             raise ValueError("`tmpdir` must be set")
         return self.tmpdir / filename
 
-    def get_document_file(self, document: ApiDocument) -> Optional[Path]:
-        logging.debug(f"Getting file. {document=}")
+    def get_document_audio(self, document: ApiDocument) -> Optional[BytesIO]:
```

Comment:
Do we want to use `Optional[T]` or `T | None`?
(I am fine either way, but a consistent style is probably nice)

(Are there linters that allow custom lint rules for stuff like this?)

Reply:
I dont know but I think our codebase does not use the | syntax anywhere yet.
I also think that it is the only python 3.10 feature we would require.

Reply:
I'll keep optional for now, but am open to changing it at some point if we agree to use `|` in general